source/MaterialXContrib/CMakeLists.txt

```diff
@@ -1,8 +1,3 @@
-include_directories(    
-    ${EXTERNAL_INCLUDE_DIRS}
-    ${CMAKE_CURRENT_SOURCE_DIR}/../
-)
```

Comment:
Better to use "target_include_directories" as this allows information about header files to be written into the CMake package file.